share/instruments/instruments.xml

```diff
@@ -7560,19 +7559,15 @@
                         <name>Closed Hi-Hat</name>
                   </Drum>
                   <Articulation>
-                        <gateTime>100</gateTime>
                         <velocity>100</velocity>
+                        <gateTime>100</gateTime>
                   </Articulation>
                   <Articulation name="sforzato">
                         <velocity>130</velocity>
                   </Articulation>
                   <Articulation name="marcato">
                         <velocity>500</velocity>
                   </Articulation>
-                  <channel>
-                        <controller ctrl="0" value="1"/>
-                        <program value="56"/> <!--Marching Snare-->
-                  </channel>
```

Comment:
An instrument without any channel? I don't think this is right and works

Reply:
It is because this one uses lowercase `<channel>` in the old instruments.xml rather than uppercase `<Channel>` like most of the others. I will need to fix it manually.

Reply:
It has been fixed and moved a few lines above.